src/components/Modal/Modal.js

```diff
@@ -1,28 +1,49 @@
 import AddCarForm from '../AddCarForm/AddCarForm';
-import StyledDivs from '../../../styles/Divs.styled';
 import Typography from '../../../styles/Typography';
-import StyledButtons from '../../../styles/Buttons.styled';
+import StyledButton from '../../../styles/StyledButton';
+import StyledFlex from '../../../styles/StyledFlex';
 
 function AddCarModal(props) {
 	if (!props.show) {
 		return null;
 	}
 	return (
-		<StyledDivs variant="modal" onClick={props.onClose}>
-			<StyledDivs variant="modalContent" onClick={event => event.stopPropagation()}>
-				<StyledDivs variant="modalHeader">
-					<Typography variant="h4">Fahrzeug hinzufügen</Typography>
-				</StyledDivs>
-				<StyledDivs variant="modalBody">
+		<StyledFlex
+			justifyContent="center"
+			alignItems="center"
+			position="fixed"
+			top="0"
+			right="0"
+			bottom="0"
+			left="0"
+			background="var(--transparent)"
+			onClick={props.onClose}
+		>
+			<StyledFlex
+				width="85%"
+				height="auto"
+				margin="20px"
+				borderRadius="8px"
+				boxShadow="var(--box-shadow)"
+				justifyContent="center"
+				alignItems="center"
+				onClick={event => event.stopPropagation()}
```

Comment:
I don't see the reason why you would need this here since there is no onClick event somewhere in sight 🧐

Reply:
adjusted according to feedback

Reply:
I readded the onClick since the form was otherwise disconnected form the modal.